cpp/test/Ice/retry/Client.cpp

```diff
@@ -32,7 +32,7 @@ Client::run(int argc, char** argv)
 
     initData.properties->setProperty("Ice.RetryIntervals", "0 1 10 1");
     installTransport(initData);
-    Ice::CommunicatorHolder ich1 = Ice::initialize(argc, argv, initData);
+    Ice::CommunicatorHolder ich1 = initialize(initData);
```

Comment:
We should use the helper, this has additional logic for test running in a remote process (iOS controller).